ClassLibrary/ClassLibrary.Test/UnitTest1.cs

```diff
@@ -6,17 +6,41 @@ namespace ClassLibrary.Test
     public class UnitTest1
     {
         [Test]
-        public void TestBooksExist()
+        public void ListBooksExist()
         {
-            Library l = new Library();
-            Assert.AreEqual(true, l.Books!=null);
+            Library lib = new Library();
+            Assert.AreEqual(true, lib.Books!=null);
         }
 
         [Test]
-        public void TestBooks()
+        public void ListIsNoBooks()
         {
-            Library l = new Library();
-            Assert.AreEqual(true, l.Books?.Count>=0);
+            Library lib = new Library();
+            Assert.AreEqual(0, lib.Books?.Count);
         }
+
+        [Test]
+        public void ListAuthorsExist()
+        {
+            Library lib = new Library();
+            Assert.AreEqual(true, lib.Authors != null);
+        }
+
+        [Test]
+        public void ListIsNoAuthors()
+        {
+            Library lib = new Library();
+            Assert.AreEqual(0, lib.Authors?.Count);
+        }
+
+        [Test]
+        public void AddNewAuthor()
```

Comment:
You need to think about all test cases, not only the happy flow:
 - what happens when the author name or last name is null
 - what happens when there's already an author with the same name, last name, or both?
 - what happens when we add multiple authors in a row?
 - do we add only one author per call?